source/08-sushi/index.rst

```diff
@@ -7,7 +7,7 @@
 SUSHI for Automated Report Harvesting
 =====================================
 
-Report providers MUST support automatic harvesting of COUNTER reports via the COUNTER_SUSHI API. The specification for the RESTful COUNTER_SUSHI API is maintained by COUNTER on SwaggerHub:
+The Standardized Usage Statistics Harvesting Initiative (SUSHI) protocol is designed to simplify the gathering of usage statistics by librarians, and report providers MUST support automatic harvesting of COUNTER reports via the COUNTER_SUSHI API. The specification for the RESTful COUNTER_SUSHI API is maintained by COUNTER on SwaggerHub:
```

Comment:
We've decided to not use "Standardized Usage Statistics Harvesting Initiative" for the R5 SUSHI, since when you look for information about this you most likely end up on the NISO page which is confusing, instead SUSHI for R5 is short for COUNTER_SUSHI API (see the SUSHI glossary entry).

report consumers instead of librarians?